sharness/t0031-ctl-add.sh

```diff
@@ -50,8 +55,9 @@ test_expect_success IPFS,CLUSTER "add files locally and compare with ipfs" '
     ipfsCmd add --quiet -r /tmp/testFolder >> cidsipfs.txt
     ipfsCmd add --quiet -r -w /tmp/testFolder >> cidsipfs.txt
 
-    ipfsCmd add --quiet --cid-version 1 -r /tmp/testFolder >> cidsipfs.txt
-    ipfsCmd add --quiet --hash sha3-512 -r /tmp/testFolder >> cidsipfs.txt
+    # uncomment when ipfs outputs base32 by default (or there is an option)
+    # ipfsCmd add --quiet --cid-version 1 -r /tmp/testFolder >> cidsipfs.txt
+    # ipfsCmd add --quiet --hash sha3-512 -r /tmp/testFolder >> cidsipfs.txt
```

Comment:
I am not sure why we need this change. The test passes and ipfs-cluster's cids are matching cids by ipfs.

Reply:
LGTM except above comment

Reply:
test breaks for cid-version=1 and --raw-leaves because ipfs is not updating their docker container since 21 days ago.